Tree_Postorder_Traversal/Tree_Postorder_Traversal.kt

```diff
@@ -0,0 +1,54 @@
+import java.util.*
+
+class Node<Int>(
+    var key:Int,
+    var left:Node<Int>? = null,
+    var right:Node<Int>? = null  
+)
+{
+    fun postorderTraversal()
+    {
+      left?.postorderTraversal()
+      right?.postorderTraversal()
+      print("$key ")
+    }
+}
+
+fun main() {
+    var read = Scanner(System.`in`)
+    println("Enter the size of Array:")
+    val arrSize = read.nextLine().toInt()
+    var arr = IntArray(arrSize)
+    val nodes = mutableListOf<Node<Int>>()
+    println("Enter the array respresentaion of binary tree")
+    
+    for(i in 0 until arrSize)
+    {
+        arr[i] = read.nextLine().toInt()
+        nodes.add(Node(arr[i]))
+    }
+    
+    for(i in 0..arrSize-2)
```

Comment:
please keep space around operator

Reply:
Can you please check now? I will squash it from next time